### ISSUE TITLE
ListView Command Set on modern list or library in classic site not displaying.

### ISSUE THREAD

**Issue by darrenparkinson** [SharePoint/sp-dev-docs#2429]
#### Category
- [ ] Question
- [ ] Typo
- [X] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior

A ListView Command Set should appear on a modern document library or list that exists in a classic site collection.

#### Observed Behavior

The ListView Command set does not appear on a modern library or list that exists in a classic site collection.

#### Steps to Reproduce

Instructions followed as per the [Build your first ListView Command Set extension document](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/extensions/get-started/building-simple-cmdset-with-dialog-api)

After issuing the `gulp serve --nobrowser` command and adding the required query parameters, the commands do not appear on a list or library that exists in a classic site.  

The commands appear when using the same parameters with a modern team site.

Thanks.

**Comment by Seiyasu** [comment]
There is a known bug with list and library extension support in the classic experiences. These only work currently in context of modern team sites, also known as group associated team sites. Work is being done to address this issue.
https://docs.microsoft.com/en-us/sharepoint/dev/spfx/extensions/overview-extensions

**Comment by darrenparkinson** [comment]
You've just repeated what’s on the overview. It’s still a bug though that is still outstanding so why would you close this? Seems to have been outstanding since at least March? What are the details of the bug? What are the timelines for a fix? I have an extension ready to deploy as soon as this bug is fixed? Please reconsider reopening until it’s fixed or pointing me to a duplicate bug report? Many thanks.

**Comment by VesaJuvonen** [comment]
Few things here. SharePoint Framework works in any site type as long as the experience is using modern experiences... so modern lists or libraries are supported with SharePoint Framework in classic sites. Site template has no impact on this. We did release a new version of the tutorials last week as part of the 1.6, which are using serve.json for ensuring query parameters in URL. I just tested this in normal E3 tenant without any issues for both custom list and a custom library. I would get the latest version of SPFx (should not matter in this case) and then follow updated tutorials to use them. Please do follow up if the updated tutorial does not work, but as I just tested this in a normal site, will close this... will though follow up on any response as needed. Thx.

**Comment by darrenparkinson** [comment]
Hi Vesa, yes, that's why I asked really -- I expected the SPFX component to work in my classic sites, so long as I was using a new library, and then saw the "bug" noted.  Currently using 1.5.1 so will attempt the upgrade and let you know how I get on.

**Comment by VesaJuvonen** [comment]
Actual version does not matter as this should have worked with 1.5.1 in the same way. Have a look on updated tutorial guidance related on using server.json for debugging - should resolve your issue.

**Comment by darrenparkinson** [comment]
Thanks Vesa.  I managed to upgrade to 1.6.0 finally.  Updated the serve.json file (note: there are some discrepancies in the doc as to whether that's serve.json or server.json).

Still no luck I'm afraid.  I can see that it is GETting the localhost files, but never seems to run the onInit or onListViewUpdated functions.

In the console I get a lot of errors relating to duplicate registration from office-ui-fabric.  I also get a couple of errors from spoapp.mini -- are they likely to cause me a problem?  Thanks again.
